docs/content/doc/advanced/customizing-gitea.en-us.md

```diff
@@ -100,6 +100,42 @@ Apart from `extra_links.tmpl` and `extra_tabs.tmpl`, there are other useful temp
 - `body_outer_post.tmpl`, before the bottom `<footer>` element.
 - `footer.tmpl`, right before the end of the `<body>` tag, a good place for additional Javascript.
 
+#### Example: Mermaid.js
+
+If you would like to add [mermaid.js](https://mermaid-js.github.io/) support to Gitea's markdown you simply add:
+
+```html
+<script src="https://unpkg.com/mermaid@8.4.5/dist/mermaid.min.js"></script>
+<!-- or wherever you have placed it -->
```

Comment:
```suggestion
<!-- (or wherever you have placed it) -->
```

Sorry, afterthought.

Reply:
it's already a parenthetical as it's in a comment. I'm not sure that parentheses add much.